RevenueCatUI/Views/TimelineView.swift

```diff
@@ -0,0 +1,322 @@
+                return .black
+            } else {
+                return UIColor(Colors.link)
+            }
+        }))
+        static let iconBackground = Color(uiColor: UIColor(dynamicProvider: { trait in
+            if trait.userInterfaceStyle == .light {
+                return .white
+            } else {
+                return .secondarySystemBackground
+            }
+        }))
+    }
+    
+    let stepConfigurations: [TimelineStepView.Configuration]
+    
+    @Environment(\.horizontalSizeClass)
+    var horizontalSizeClass
+    var axis: NSLayoutConstraint.Axis {
+        horizontalSizeClass == .regular ? .vertical : .horizontal
+    }
+    
+    var body: some View {
+        if axis == .horizontal {
+            HStack(spacing: 0) {
+                content
+            }
+        } else {
+            VStack(alignment: .leading, spacing: 0) {
+                content
+            }
+        }
+    }
+    
+    @ViewBuilder
+    var content: some View {
+        ForEach(stepConfigurations) {
+            TimelineStepView(configuration: $0, axis: axis.opposite)
+        }
+    }
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+struct TimelineStepView: View {
+    
+    struct Configuration: Identifiable {
+        let title: String
+        let icon: String
+        let iconBackgroundColor: Color
+        let iconForegroundColor: Color
+        let subtitle: String
+        let linkColor: Color
+        let linkPosition: LinkPosition
+        
+        init(
+            title: String,
+            icon: String,
+            iconBackgroundColor: Color = TimelineView.Colors.iconBackground,
+            iconForegroundColor: Color = .primary,
+            subtitle: String, 
+            linkColor: Color = TimelineView.Colors.link,
+            linkPosition: LinkPosition
+        ) {
+            self.title = title
+            self.icon = icon
+            self.iconBackgroundColor = iconBackgroundColor
+            self.iconForegroundColor = iconForegroundColor
+            self.subtitle = subtitle
+            self.linkColor = linkColor
+            self.linkPosition = linkPosition
+        }
+        
+        var id: String {
+            title + subtitle
+        }
+    }
+    
+    enum LinkPosition {
+        case leading
+        case trailing
+        case both
+    }
+    
+    let configuration: Configuration
+    let axis: NSLayoutConstraint.Axis
+    let iconSize: CGFloat = 32
+    
+    var body: some View {
+        if axis == .vertical {
+            ZStack {
+                link
+                content
+            }
+            .frame(minWidth: 100)
+        } else {
+            ZStack(alignment: .init(horizontal: .iconAlignment, vertical: .center)) {
+                link
+                content
+            }
+            .frame(minHeight: 50)
+            .padding([.leading, .trailing], 20)
```

Comment:
since Timeline-Testimonials-CompanyLogos all have the same offsets,
lets do this at the top level (LinTemplate4).
Remove this.